Week1/homework/world-queries.txt

```diff
@@ -37,4 +37,209 @@ SELECT city.Name FROM city JOIN country ON city.CountryCode = country.Code WHERE
+| Karachi          |    9269265 |
+| Istanbul         |    8787958 |
+| Ciudad de México |    8591309 |
+| Moscow           |    8389200 |
+| New York         |    8008278 |
++------------------+------------+
+10 rows in set (0.00 sec)
+
+6.Select the top 3 countries by population that their names start with the letter ‘P’
+
+SELECT name, population FROM country WHERE name like 'P%' ORDER BY population DESC LIMIT 3;
++-------------+------------+
+| name        | population |
++-------------+------------+
+| Pakistan    |  156483000 |
+| Philippines |   75967000 |
+| Poland      |   38653600 |
++-------------+------------+
+3 rows in set (0.00 sec)
+
+7.What is the population in Asia?
+
+SELECT sum(population) FROM country WHERE continent = 'Asia';
++-----------------+
+| sum(population) |
++-----------------+
+|      3705025700 |
++-----------------+
+1 row in set (0.01 sec)
+
+8.Which languages are spoken in the Region of ‘South America’
+
+SELECT Language, CountryCode FROM CountryLanguage WHERE CountryCode IN (SELECT Code FROM Country WHERE Region = 'South America');
++------------------+-------------+
+| Language         | CountryCode |
++------------------+-------------+
+| Indian Languages | ARG         |
+| Italian          | ARG         |
+| Spanish          | ARG         |
+| Aimará           | BOL         |
+| Guaraní          | BOL         |
+| Ket?ua           | BOL         |
+| Spanish          | BOL         |
+| German           | BRA         |
+| Indian Languages | BRA         |
+| Italian          | BRA         |
+| Japanese         | BRA         |
+| Portuguese       | BRA         |
+| Aimará           | CHL         |
+| Araucan          | CHL         |
+| Rapa nui         | CHL         |
+| Spanish          | CHL         |
+| Arawakan         | COL         |
+| Caribbean        | COL         |
+| Chibcha          | COL         |
+| Creole English   | COL         |
+| Spanish          | COL         |
+| Ket?ua           | ECU         |
+| Spanish          | ECU         |
+| English          | FLK         |
+| Creole French    | GUF         |
+| Indian Languages | GUF         |
+| Arawakan         | GUY         |
+| Caribbean        | GUY         |
+| Creole English   | GUY         |
+| Aimará           | PER         |
+| Ket?ua           | PER         |
+| Spanish          | PER         |
+| German           | PRY         |
+| Guaraní          | PRY         |
+| Portuguese       | PRY         |
+| Spanish          | PRY         |
+| Hindi            | SUR         |
+| Sranantonga      | SUR         |
+| Spanish          | URY         |
+| Goajiro          | VEN         |
+| Spanish          | VEN         |
+| Warrau           | VEN         |
++------------------+-------------+
+42 rows in set (0.00 sec)
+
+9.What are the languages spoken on all cities named ‘Barcelona’ (you may need to join 3 tables =) )
+
+with join :
+ SELECT Language from CountryLanguage cl JOIN Country co ON cl.CountryCode = co.Code
```

Comment:
nice triple join ! but the inner data scientist in me is wondering why we got spanish twice 🤔 , think you can print the city name and country name as well ?

Reply:
Sure I'll try that 👍 , as for the ERD, I'm working on it and will push it today along with the bonus assignment , I just didn't know you will review the homework already 💃